Libplanet/Blockchain/Policies/IBlockPolicy.cs

```diff
@@ -43,6 +43,6 @@ DateTimeOffset currentTime
         /// followed by a new <see cref="Block{T}"/> to be mined.</param>
         /// <returns>A right <see cref="Block{T}.Difficulty"/>
         /// for a new <see cref="Block{T}"/> to be mined.</returns>
-        int GetNextBlockDifficulty(IEnumerable<Block<T>> blocks);
+        int GetNextBlockDifficulty(IReadOnlyList<Block<T>> blocks);
```

Comment:
Changes in this file also break API backward compatibility, we need to write the corresponding changelogs.